backend/tessen/config.go

```diff
@@ -0,0 +1,34 @@
+package tessen
+
+import (
+	"fmt"
+)
+
+const (
+	// LowerBound is the minimum frequency that tessen will phone home
+	LowerBound = 1
+
+	// UpperBound is the maximum frequency that tessen will phone home
+	UpperBound = 360
+
+	// DefaultTessenFrequency is the default frequency at which tessen will phone home
+	DefaultTessenFrequency = 1
+)
+
+// Config is the tessen configuration
+type Config struct {
```

Comment:
I might have missed some discussions around that so I apologize in advance but my understanding was that we would create a new API group for Tessen, so I expected this to live in a folder like `api/tessen/v2` and see the Tessen API routes under the same path, which isn't the case.

Reply:
You're totally right, we initially discussed and spec'd a versioned API. We had another small conversation regarding leader election and some other tessen details. We concluded versioning might be a little overkill, since it's not necessarily a consumer resource, and right now we only let the user determine opt-out status. Very similar to the `/health` endpoint. Do you have any strong feelings for versioning it, because we can def discuss it further!

Reply:
I'm kind of on the fence. I agree that if we only let users determine opt-out status, it's definitively overkill. I guess it ultimately depends on what the users can control; are we going to allow customization of the lower and upper bound frequencies? What about the HTTP proxy stuff mentioned in the specs? If so, we are limiting configuration to the `sensuctl tessen` subcommand (in contrast to `sensuctl create`).

Also, it might not be important but it prevents targeting the Tessen config in RBAC; you'll need access to everything to manage non-resources endpoints like `/tessen`, which is probably fine, but just something to keep in mind.

Reply:
I feel that for sure. Right now, the only thing we're letting the user determine is opt-out status, but if in the future if we do need to add fields, it might better if we already have versioning concepts in place. I assumed tessen would not be gated by RBAC, but we never explicitly discussed that so I could be wrong. @portertech @echlebek do you have any thoughts to add on all of this?

Reply:
I think only admins should be able to enable and disable tessen.

Reply:
This has all been resolved by versioning the tessen api 😄